ansible_navigator/utils.py

```diff
@@ -115,6 +115,34 @@ def environment_variable_is_file_path(
     return messages, errors, file_path
 
 
+def find_ansible_config_file() -> Tuple[List[LogMessage], List[str], Optional[str]]:
+    """Load config file(first found is used): CWD, HOME, /etc/..."""
+
+    messages: List[LogMessage] = []
+    errors: List[str] = []
+    path = None
+    filename = "ansible.cfg"
+
+    # Current working directory
+    potential_paths = [abs_user_path(os.path.join(os.getcwd(), filename))]
```

Comment:
The Ansible config file can be set using the ANSIBLE_CONFIG env variable also. Another option to get the config file is to execute the `which ansible` command using runner `run_command` API and fetch the config_file path value.

Reply:
removed this logic entirely